src/viur/core/modules/file.py

```diff
@@ -1315,3 +1316,19 @@ def start_delete_pending_files():
         .filter("pending =", True)
         .filter("creationdate <", utils.utcNow() - datetime.timedelta(days=7))
     )
+
+
+# DEPRECATED ATTRIBUTES HANDLING
+
+def __getattr__(attr: str) -> object:
+    if entry := {
+            # stuff prior viur-core < 3.7
+            "GOOGLE_STORAGE_BUCKET": ("File.get_bucket()", _private_bucket),
+    }.get(attr):
+        func = entry[1]
+        msg = f"{attr} was replaced by {entry[0]}"
```

Comment:
We could simplify this, but this works of course too 
```suggestion
        msg = f"{entry[1]} was replaced by {entry[0]}"
```

Reply:
That change is ok for me, let's move forward to merge this please